anndata/tests/helpers.py

```diff
@@ -322,6 +349,18 @@ def assert_equal_h5py_dataset(a, b, exact=False, elem_name=None):
     assert_equal(b, a, exact, elem_name=elem_name)
 
 
+@assert_equal.register(DaskArray)
+def assert_equal_dask_array(a, b, exact=False, elem_name=None):
+
+    from dask.array.utils import assert_eq
+
+    if exact:
+        assert_eq(a, b, check_dtype=True, check_type=True, check_graph=False)
+    else:
+        # TODO: Why does it fail when check_graph=True
```

Comment:
After reading the assert_equal function I think that options checks if the computation graphs are overlapping or not because it calls the following method inside
```python
def _check_dsk(dsk):
    """Check that graph is well named and non-overlapping"""
```
Then I realized .copy() does nothing for dask arrays. I could copy the computation graph (see https://github.com/dask/dask/issues/6674 and https://docs.dask.org/en/latest/graph_manipulation.html#api) but I don't know if I should.

Reply:
I don't think we need to do that here